synapse/federation/sender/per_destination_queue.py

```diff
@@ -154,7 +154,7 @@ def pending_edu_count(self) -> int:
             + len(self._pending_edus_keyed)
         )
 
-    def send_pdu(self, pdu: EventBase) -> None:
+    def send_pdu(self, *pdus: EventBase) -> None:
```

Comment:
this should probably be called `send_pdus`. Also note that the docstring refers to *a* PDU.